lib/go-atscfg/remapdotconfig_test.go

```diff
@@ -8075,13 +8073,13 @@ func TestMakeRemapDotConfigStrategies(t *testing.T) {
 			Name:       "serverpkgval",
 			ConfigFile: "package",
 			Value:      "serverpkgval __HOSTNAME__ foo",
-			Profiles:   []byte(*server.Profile),
+			Profiles:   []byte(`["` + server.ProfileNames[0] + `"]`),
```

Comment:
Previously, this was a string containing just the Profile name. It's now surrounded by square brackets and quotes - why?

Reply:
Because the profile name is an array and resolving only the first element.

Reply:
Profiles was already an array in [Parameter responses](https://traffic-control-cdn.readthedocs.io/en/latest/api/v2/parameters.html#response-structure)

Reply:
ok, it takes json a RawMessage.. so i needn't do anything..copy.